sql/V003__Add_Root_User.sql

```diff
@@ -0,0 +1,7 @@
+
+-- Set the root user's password to '' if it hasn't been set. This will allow the
+-- first administrator to log in.
+UPDATE gem_user
+SET    md5 = md5('')
+WHERE  id  = 'root'
+AND    md5 = ''
```

Comment:
I like flyway but I wonder if we shouldn't just put everything in one big initialization file until the first release.  After that, migrations will obviously be necessary.